corpus/types.txt

```diff
@@ -198,8 +198,10 @@ type G = H[A1] {
   def foo[A2 <: Stepper[_]]: A2
 }
 
-type I = F1[A1:
-  def foo: A2]
+type I = F1{
+  def foo: A}
```

Comment:
I added this test to demonstrate the use of braceless syntax, in a type parameter position, outdented by a closing bracket. Not being able to parse it seems like a regression, assuming that what I wrote is in fact a legal Scala.

Reply:
Hm, could you please proved a compiling Scala example with such code?
e.g.
```scala
object X:
  trait F1[T]
  type I = F1[A1:
    def foo: A2]
```
would fail to compile

Reply:
```scala
scala> type I = List[AnyRef:
     |   def toString: String]
// defined alias type I = List[AnyRef{def toString: String}]
```

Here's I guess a `List` type whose element must be a structural type that implements `toString`

Reply:
Thanks!
I had scalac 3.1.2, which was old enough not to compile this

Reverted this change